packages/auto-label/src/auto-label.ts

```diff
@@ -246,7 +246,7 @@ handler.autoLabelOnPR = async function autoLabelOnPR(
       logger.info(
         `Path label added to PR #${pull_number} in ${owner}/${repo} is ${path_label}`
       );
-      new_labels.push(path_label);
+      new_labels = path_label;
```

Comment:
I like const, because it's easier to reason about the code.  I'd prefer to leave line 238 as const, and change line 249 to 
```new_labels.push(...path_label);```